include/sys/uberblock_impl.h

```diff
@@ -54,6 +55,10 @@ struct uberblock {
 
 	/* highest SPA_VERSION supported by software that wrote this txg */
 	uint64_t	ub_software_version;
+
+	/* Maybe missing in uberblocks we read, but always written */
+	uint64_t	ub_mmp_magic;	/* MMP_MAGIC			*/
+	uint64_t	ub_mmp_delay;	/* nanosec since last MMP write	*/
```

Comment:
I opened a PR, please take a look: https://github.com/openzfs/openzfs/pull/425

If that goes well, we just need to make sure that whoever upstreams first, merges the other OS's commit. For example, if MMP gets pushed to ZoL first, we need to make sure to get it to illumos/OpenZFS before upstreaming the zpool checkpoint.